compiler/minmax-embedder/requires.cmake

```diff
@@ -1,2 +1,5 @@
+require("arser")
 require("loco")
 require("luci")
+require("safemain")
+require("vconone")
```

Comment:
This change forces to build `arser`, `safemain`, and `vonone` when we want to build library only and don't need unittest and driver. 
When we want to build library only for runtime's ODC, we will meet this issue. 
To avoid this issue, we should divide library and driver project, like other compiler projects.

Reply:
Above comment is my misunderstand. Forget that.